playbooks/tasks/portal-configs-get.yml

```diff
@@ -23,53 +23,40 @@
   loop: "{{ webportal_server_config_defaults | dict2items }}"
   no_log: True
 
-- name: Set MongoDB flag
-  set_fact:
-    portal_mongodb_on: >-
-      {{ webportal_server_config.portal_modules is defined and
-      webportal_server_config.portal_modules is not none and
-      ('m' in webportal_server_config.portal_modules or 'a' in webportal_server_config.portal_modules) }}
```

Comment:
Where did this go? Shouldn't we still be checking the portal modules?

Reply:
Mongo is mandatory for all portals, Skyd doesn't start without Mongo replicaset so we dont need `portal_mongodb_on` flag anymore.